azblob/reader.go

```diff
@@ -67,23 +65,16 @@ func NewReaderNoAuth(accountName string, url string, container string) (Reader,
 		nil,
 	)
 	if err != nil {
-		logger.Sugar.Infof("unable to create serviceclient %s: %v", url, err)
 		return nil, err
```

Comment:
For future work: propagating an error from  a third party service like this is not considered good practice. Usually one defines an error type, wraps the error with '%v' and returns that.

This is just a comment...

Reply:
agreed! 

We should have a TD ticket to address this, especially in our common go repo :)